chaoscenter/web/src/controllers/ActiveProjectMemberList/helper.ts

```diff
@@ -0,0 +1,20 @@
+import type { ProjectMember } from '@api/auth';
+
+export function generateActiveMemberTableContent(memberData: Array<ProjectMember> | undefined): Array<ProjectMember> {
```

Comment:
I don't understand the utility of this function...it's not doing any modification on memberData